plugins/generator-1.16.4/forge-1.16.4/templates/block.java.ftl

```diff
@@ -108,8 +108,14 @@ public class ${name}Block extends ${JavaModName}Elements.ModElement {
 					BiomeColors.getGrassColor(world, pos) : GrassColors.get(0.5D, 1.0D);
 				<#elseif data.tintType == "Foliage">
 					BiomeColors.getFoliageColor(world, pos) : FoliageColors.getDefault();
-				<#else>
+				<#elseif data.tintType == "Water">
 					BiomeColors.getWaterColor(world, pos) : -1;
+				<#elseif data.tintType == "Sky">
+					Minecraft.getInstance().world.getBiomeManager().getBiome(pos).getSkyColor() : 8562943;

```

Comment:
Any specific reason you use world from Minecraft here instead of the world from the lambda?

Reply:
The world from the lambda isn't an instance of IWorldReader, so I can't use it to get the biome

Reply:
Yep, just noticed this :)